.github/.license_check.ini

```diff
@@ -0,0 +1,29 @@
+# Authorized and unauthorized licenses in LOWER CASE
+[Licenses]
+authorized_licenses:
+        3-clause bsd
+        apache
+        apache 2.0
+        apache software
+        apache software license
+        apache-2.0
+        bsd
+        bsd license
+        new bsd
+        new bsd license
+        simplified bsd
+        gnu lgpl
+        isc license
+        isc license (iscl)
+        lgpl with exceptions or zpl
+        mit
+        mit license
+        mozilla public license 2.0 (mpl 2.0)
+        python software foundation
+        python software foundation license
+        zpl 2.1
+        GNU Lesser General Public License v2 or later (LGPLv2+)
+        MPL-2.0
+
+unauthorized_licenses:
+        gpl v3
```

Comment:
add newline at the end. and for the license, i think it is better to sort them in order.